deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -357,14 +357,25 @@ def callback_detect_file_deleted_event(line):
     return None
 
 
-def callback_restricted(line):
-    """ Callback that detects if a line in a log  if a file is ignored due to configured restrict tag.
+def callback_audit_cannot_start(line):
+    """ Callback that detects if a line shows whodata engine could not start and monitoring switched to realtime.
 
     Args:
         line (String): string line to be checked by callback in FileMonitor.
 
     Returns:
-        returns the entry that is being ignored.
+        Returns True if line matches.

```

Comment:
Apply this suggestion to all occurrences.

```suggestion
        boolean: True if the line matches None otherwise.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3939/commits/f56f61577f3310474fbdac39dcf3a2f8401f2a8c